### DISCUSSION TITLE
How can I mount Google Cloud Bucket Storage?

### DISCUSSION THREAD

**Discussion by ralflukner** [iterate-ch/cyberduck#16314] [Q&A]
### Discussed in https://github.com/iterate-ch/cyberduck/discussions/16310

<div type='discussions-op-text'>

<sup>Originally posted by **ralflukner** September 11, 2024</sup>
Do you know how I can mount Google Cloud Bucket Storage?  I used to have this working but lost the configuration.  I cannot recreate it in working condition (I previously used an AWS S3 configuration).
Thank you,
--Ralf</div>

**Comment by dkocher** [comment]
Refer to our [documentation](https://docs.cyberduck.io/protocols/googlecloudstorage/).

**Reply by ralflukner** [reply]
Thank you 😃

But the method in the documentation for interoperable does not seem to work:

[Interoperable Access](https://docs.cyberduck.io/protocols/googlecloudstorage/#id2)[](https://docs.cyberduck.io/protocols/googlecloudstorage/#interoperable-access)
For interoperable access, please use an S3-compatible protocol (like Amazon S3, not Google Storage) with the server set to storage.googleapis.com.
This is not possible because changing the server to storage.googleapis.com changes the protocol to WebDAV.  Google Storage is another protocol that is allowed.
Using the Google Storage Protocol works initially...
Login Google Storage - LMC
Open web browser to authenticate and obtain an authorization code.
(login with web browser) -->> Open Mountain Duck
https://accounts.google.com wants to open this application. (Mountain Duck)
(Choose Open Mountain Duck)
***Connection failed (Google Storage - LMC)***
Broken pipe. The connection attempt was rejected. The server may be down, or your network may not be properly configured. (Disconnect) (Try again)

![image](https://github.com/user-attachments/assets/913397f8-4421-4abe-93f6-001471076853)

**Reply by dkocher** [reply]
For _Interoperable Access_ you will have to connect using the _Amazon S3_ option.

**Reply by tamka** [reply]
Last time I couldn't connect to a Google Cloud Storage bucket, I also saw the warning triangle icon beside the URL line. In my case, it turned out that I was connected to a network that wasn't allowing traffic to that URL. When I connected the computer to a different network, it worked fine. So you might want to check if there's a rule on that specific network that is blocking your access.

**Comment by ralflukner** [comment]
I try to but that forces the URL to be s3.amazon….

**Reply by dkocher** [reply]
Install the generic _S3 (HTTPS)_ connection profile from _Preferences → Profiles_.

**Reply by ralflukner** [reply]
I get an error when I choose that ... (as well as no choices such as a generic S3 (HTTPS) connection profile)

```
Listing directory / failed. (Amazon S3)
The requested action cannot be performed with a non-authenticated service: List all buckets. Please contact your web hosting service provider for assistance.
(Cancel)  (Try Again)
```

**Reply by dkocher** [reply]
Change the _Server_ to `storage.googleapis.com` instead.
